multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinder.java

```diff
@@ -84,10 +84,8 @@
             throw new IllegalStateException(branch.getSourceId() + " not found");
         }
         SCMHead head = branch.getHead();
-        SCMRevision tip = scmSource.fetch(SCMHeadObserver.select(head), listener).result();
+        SCMRevision tip = scmSource.fetch(head, listener);
         if (tip == null) {
```

Comment:
Well, if the `tip` is null, something is wrong, which we would like to diagnose somehow.